ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -84,3 +84,9 @@ def eye(N, M=None, k=0, dtype=None):
 @to_ivy_arrays_and_back
 def triu(m, k=0):
     return ivy.triu(m, k=k)
+
+
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+def empty(shape, dtype=None):
+    return ivy.empty(shape, dtype=dtype)
```

Comment:
Please add a new line character after this line. This currently causes lint error:
`W292 no newline at end of file`

Reply:
Done. Added a new line character.